helm/cluster-api-provider-aws/templates/eks-bootstrap-components.yaml

```diff
@@ -0,0 +1,266 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: bootstrap-eks
+    control-plane: controller-manager
+  name: capa-eks-bootstrap-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    ${AWS_CONTROLLER_IAM_ROLE/#arn/eks.amazonaws.com/role-arn: arn}
```

Comment:
Not sure if we have this IAM role. Can we remove this annotation? not sure how we do this in our aws-operator